api/rpc/stack/stack.go

```diff
@@ -7,34 +7,37 @@ import (
 	"golang.org/x/net/context"
 )
 
-// Stack is used to implement stack.StackServer
-type Stack struct {
+// Server is used to implement stack.Server
+type Server struct {
```

Comment:
I definitely don't prefer calling it `Server`. Conflict with some other type, like a message?